introduction.md

```diff
@@ -4,4 +4,11 @@ Hello my name is L|0 and I am trying to complete the CodeSchool "Mastering GitHu
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?